src/markdown-pages/docs/ecosystem/ecosystem-role-based-access.md

```diff
@@ -0,0 +1,208 @@
+| CPS_PROPERTIES_DELETE | Allows the deletion of CPS properties |
+| CPS_PROPERTIES_SET | Set a CPS property value |
+| GENERAL_API_ACCESS | Be able to perform basic actions on the system via the REST api |
+| RUNS_DELETE_OTHER_USERS | Allows the user to delete runs that other users submitted |
+| SECRETS_DELETE | Allows the deletion of secrets |
+| SECRETS_GET_UNREDACTED_VALUES | Allows you to get secrets which contain the real secret value, rather than a redacted value |
+| SECRETS_SET | Allows setting of secrets |
+| USER_EDIT_OTHER | Allows the editing of a different user on the system. This includes changing their role, deleting personal access tokens, or deleting the user completely |
+
+
+## Available Roles
+This table shows the list of built-in roles:
+| Role name | Description | Actions someone with this role can perform |
+|-----------|-------------|--------------------------------------------|
+| deactivated | A user who has no permissions at all, and is able to do nothing on the Galasa service | none |
+| tester | A user who writes and runs tests, looks at the test results and diagnoses errors in the system under test | GENERAL_API_ACCESS |
+| admin     | A Galasa service administrator. Able to do any operation supported by the Galasa service | all |
+| owner       | The owner of the Galasa service. Such users have all the rights of an administrator, but nobody can delete their id or change their role without changing the kubernetes configuration | all |
+
+
+
+## Constraints
+The system also has some constraints on how these resources can be set up and used:
+- Nobody can change their own "role", assigning a different role to yourself is not permitted
+- New users on the system are assigned a default role when they first login to the Galasa service. This role is configurable. More details on how to configure this default role is described in section [how to set the default user role](#how-to-set-the-default-user-role).
+- One or more users can be assigned the `owner` role. Such users are privileged because an administrator role cannot directly update their role away from `owner` using the standard mechanisms. Similarly, such "owners" of the Galasa service cannot be removed, even by another "owner" without changing kubernetes configuration.
+- An administrator cannot delete their own user record. This constraint attempts to make sure that there is at least one owner or administrator on the system. If nobody active in your organisation has `admin` or `owner` rights, then a new owner of the service can be nominated using the Galasa service install/update instructions. See [How to nominate a user as the Galasa service `owner`](#how-to-nominate-a-user-as-the-galasa-service-owner)
+- An administrator cannot change their own role.
+- Not even administrators can change the role of a user nominated as an `owner`.
+- The `owner` role can only be assigned using the Kubernetes property, not via the command-line or REST interface.
```

Comment:
"Helm property" instead of "Kubernetes property"?

Reply:
fixed.